packages/next/next-server/server/image-optimizer.ts

```diff
@@ -370,8 +370,7 @@ function parseCacheControl(str: string | null): Map<string, string> {
   return map
 }
 
-export function getMaxAge(str: string | null): number {
-  const minimum = 60
+export function getMaxAge(str: string | null, minimum = 60): number {
```

Comment:
```suggestion
export function getMaxAge(str: string | null, minimum: number): number {
```

Let's always pass in the config value here